js-exercises/abbreviate-string/abbreviateString.js

```diff
@@ -0,0 +1,20 @@
+function abbreviateString(str) {
+  if (typeof (str) !== 'string') {
+    throw Error(`Expected string, got ${typeof str}`);
+  }
+
+  if (str.trim().length < 1) {
+    throw Error('Got empty string.');
+  }
+
+  if (str.split(' ').length === 1) {
+    return str[0].toUpperCase() + str.substring(1);
+  }
+
+  const stringArray = str.split(' ');
```

Comment:
again .trim is missing
```suggestion
  const stringArray = str.trim().split(' ');
```

Ideally, .trim() should be done in starting.
And trimmed value should be passed further